lighthouse-core/test/config/budgets-test.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Budgets = require('../../config/budgets');
+const assert = require('assert');
+/* eslint-env jest */
+
+describe('Budgets', () => {
+  let budgetsJson;
+  beforeEach(() => {
+    budgetsJson = [{
```

Comment:
super nit: it's a tad hard to tell what the JSON nesting actually is with this indentation. could run `node_modules/.bin/prettier` on it to make it easy :)